### ISSUE TITLE
bring the light should give the buff when accepting quest

### ISSUE THREAD

**Issue by connormcgregor** [elysium-project/server#3023]
http://www.wowhead.com/quest=3636/bring-the-light#comments

@thepparker By remedi (1,659 – 1·3·10) on 2007/08/19(Patch 2.1.3)

As you accept the quest, the archbishop will make you a fortitude buff.

It says you get the buff when you accept the quest.

I have noticed it is changed to give the buff when quest is completed 
But youtube video shows that he doesnt get a buff when completing.

**Comment by connormcgregor** [comment]
https://m.youtube.com/watch?v=mHJe2HpX3g4
Shows you arent given fortitude

**Comment by sadfx** [comment]
Correct. As of right now it's not sharable despite the quest giver not buffing you, which is what made the quest non-sharable in the first place.

**Comment by GuybrushGit** [comment]
Fixed with https://github.com/elysium-project/server/pull/2355/commits/1fe9bd618cde1d3c1c551a85f1e29f1770a0ad21
